webapp/app.py

```diff
@@ -150,6 +152,9 @@ def utility_processor():
 # Simple routes
 app.add_url_rule("/advantage", view_func=advantage_view)
 app.add_url_rule("/advantage/subscribe", view_func=advantage_shop_view)
+app.add_url_rule(
+    "/advantage/subscribe/thanks", view_func=advantage_thanks_view
```

Comment:
We normally use thank-you instead of thanks. Or would confirmation be a better URL?

Reply:
@matthewpaulthomas what do you think?

Reply:
Updated to /thank-you